app/locale/de-CH.coffee

```diff
@@ -2620,15 +2620,15 @@ module.exports = nativeDescription: "Dütsch (Schwiiz)", englishDescription: "Ge
 #    create_sharable_interactive_web_page: "Create a sharable interactive web page"
 
 #  anonymous_teacher:
-#    notify_teacher: "Notify Teacher"
+#    notify_teacher: "Lohs dini Lehrperson wüsse"
 #    create_teacher_account: "Create free teacher account"
-#    enter_student_name: "Your name:"
-#    enter_teacher_email: "Your teacher's email:"
-#    teacher_email_placeholder: "teacher.email@example.com"
-#    student_name_placeholder: "type your name here"
-#    teachers_section: "Teachers:"
-#    students_section: "Students:"
-#    teacher_notified: "We've notified your teacher that you want to play more CodeCombat in your classroom!"
+#    enter_student_name: "Din Name:"
+#    enter_teacher_email: "D'Email-adresse vo dinere Lehrperson:"
+#    teacher_email_placeholder: "lehrperson.email@biispiil.ch"
```

Comment:
This is an example email so we can keep it as "lehrperson.email@example.com". The @example.com email can be used for these kind of things.